app/views/core/call_us_about_a_payment_plan.scala.html

```diff
@@ -17,26 +17,27 @@
 @import partials.{call_us_generic, extra_support, before_you_call, we_are_likely_to_ask}
 
 @this(
-  main: views.html.main,
-  viewsHelpers: views.ViewsHelpers
+        main: views.html.main,
+        viewsHelpers: views.ViewsHelpers
 )
 
 @(
         isWelsh: Boolean,
-        loggedIn: Boolean = false)(implicit
+        loggedIn: Boolean = false,
+        cannotAfford: Boolean = false)(implicit
         messages: play.api.i18n.Messages,
         request: Request[_],
         appConfig: config.AppConfig
 )
 
 @main(
     title = messages("ssttp.call-us.generic.h1"),
-    hasBackLink = false
+    hasBackLink = if(cannotAfford) true else false
```

Comment:
since `cannotAfford` is a boolean this is equivalent to `hasBackLink = cannotAfford` which would be a bit more efficient